llvm/lib/Target/EraVM/EraVMInstrFormats.td

```diff
@@ -23,6 +23,18 @@ def OpndRR         : SrcOperandMode;
 def OpndIR         : SrcOperandMode;
 def OpndSR         : SrcOperandMode;
 def OpndCR         : SrcOperandMode;
+def OpndRI         : SrcOperandMode;
```

Comment:
Please mention that these forms are to support SEL pseudo and they don't describe a real EraVM addressing mode.